html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue

```diff
@@ -171,11 +171,6 @@ export default {
     scroll100
   },
   props: {
-    storeName: { // from router
```

Comment:
We actually need to specify `storeName` otherwise we through an error.

Reply:
If we don't want to define `storeName` from the router, it can be defined here:

```
    storeName: {
      type: String,
      default: "$_users"
    },
```